src/app/style.less

```diff
@@ -0,0 +1,30 @@
+
+
+@import "variables";
+@import "mixins";
+@import "grid";
+
+@import "pages/home/home";
+
+@color: #fdd;
+
+body {
+    background: @color;
+}
+
+html, body, #app, #app-component, #home-page, #list, #map-canvas { 
+    height: 100%; 
+    margin: 0; 
+    padding: 0; 
+    overflow: hidden; 
+}
+#map-canvas { 
+    width: 70%;
```

Comment:
It's worth talking about more... I haven't fully settled on what layout.less (introduced in previously mentioned later commit [in wip-update-production branch]) will ultimately look like. We could review flex box support, but I think that if you use "box-sizing: border-box" then you can get pretty far with just margin and padding... but yeah, worth more discussion.